cordapp-cpk/src/main/kotlin/net/corda/plugins/cpk/OsgiExtension.kt

```diff
@@ -61,6 +61,11 @@ open class OsgiExtension(objects: ObjectFactory, jar: Jar) {
             "org.hibernate.proxy"
         ))
 
+        private val Array<String>.packageRange: IntRange get() {
+            val firstIdx = if (size > 2 && this[0] == "META-INF" && this[1] == "versions") { 3 } else { 0 }
+            return firstIdx..size - 2
+        }
```

Comment:
it doesn't make much difference, but this is not a generic method that can apply to any array of Strings, it would be easier to understand as a private method

Reply:
> it doesn't make much difference, but this is not a generic method that can apply to any array of Strings, it would be easier to understand as a private method

You're right - it doesn't make much difference :wink:. However, I've changed it anyway.